src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -618,15 +618,55 @@ public function collectFromCode($content, $fileName, Module $module)
         $potentialClassName = null;
         $currentUse = null;
         $currentUseAlias = null;
+        $inVar = null;
```

Comment:
```suggestion
        $inVar = null; // Tracks the value currently being assigned to a variable
```
As per https://github.com/silverstripe/silverstripe-framework/pull/11521#discussion_r1903932492
Have I understood that correctly? If not please adjust the comment so it's appropriately indicating the purpose of this variable.

Reply:
it's not tracking the value, it's tracking the name of the variable. the value is stored in inVarText. i've added comments to reflect this